### ISSUE TITLE
Unable to register Log Owl event

### ISSUE THREAD

**Issue by stctheproducer** [jz222/loggy-adapter-nodejs#4]
I created an account on [Log Owl](https://logowl.io/), I created a project and got a ticket number but I am unable to connect to Log Owl and I am receiving an error. The code I am running on both my local machine and a Digital Ocean server is:

```javascript
require('dotenv').config()

const logowl = require('@logowl/adapter-nodejs')

logowl.init({ ticket: process.env.LOG_OWL_TICKET })

try {
  test()
} catch (error) {
  logowl.emitError(error)
}
```

And the error response I am receiving is:

```bash
Failed to register Log Owl event with error:

Error: ENOENT: no such file or directory, open '/demo-server/lowgowl.js'
```
Could there be something I am missing in the config?

**Comment by jz222** [comment]
Hi,

thanks for using Log Owl.

I've tried your code and it works fine. Seems like you are trying to execute a file, which doesn't exist at the given path. I created a repository here: https://github.com/jz222/logowl-nodejs-example

Clone it, install the dependencies with `npm i` and run the code with `npm start`. Make sure to replace the ticket number in the `.env` file first.

<img width="1298" alt="Screenshot 2021-01-27 at 22 38 21" src="https://user-images.githubusercontent.com/43854691/106057554-6baf4c00-60f0-11eb-90ea-20c55014aff5.png">
